ibis/omniscidb/tests/test_client.py

```diff
@@ -100,6 +100,126 @@ def test_union_op(alltypes):
         expr.compile()
 
 
+@pytest.mark.parametrize(
+    'cols_with_types',
+    [
+        {},
+        {'c': 'DOUBLE'},
+        {'c': 'DOUBLE', 'd': 'TEXT', 'e': 'POINT', 'f': 'POLYGON'},
+    ],
+)
+def test_add_column(con, cols_with_types):
+    table_name = 'my_table'
+
+    con.drop_table(table_name, force=True)
+
+    schema = ibis.schema([('a', 'float'), ('b', 'int8')])
+
+    con.create_table(table_name, schema=schema)
+
+    col_count = len(cols_with_types)
+
+    if col_count == 0:
+        isException = False
+        try:
+            con.add_column(table_name, cols_with_types)
+        except com.IbisInputError:
+            isException = True
+        finally:
+            con.drop_table(table_name)
+
+        if not isException:
+            assert False
+    elif col_count == 1:
+        con.add_column(table_name, cols_with_types)
+
+        schema_for_check = ibis.schema(
+            [('a', 'float'), ('b', 'int8'), ('c', 'double')]
+        )
+
+        try:
+            t = con.table(table_name)
+
+            for k, i_type in t.schema().items():
+                assert schema_for_check[k] == i_type
+        finally:
+            con.drop_table(table_name)
+    else:
+        con.add_column(table_name, cols_with_types)
+
+        schema_for_check = ibis.schema(
+            [
+                ('a', 'float'),
+                ('b', 'int8'),
+                ('c', 'double'),
+                ('d', 'string'),
+                ('e', 'point'),
+                ('f', 'polygon'),
+            ]
+        )
+
+        try:
+            t = con.table(table_name)
+
+            for k, i_type in t.schema().items():
+                assert schema_for_check[k] == i_type
+        finally:
+            con.drop_table(table_name)
+
+
+@pytest.mark.parametrize('column_names', [[], ['a'], ['a', 'b', 'c']])
+def test_drop_column(con, column_names):
```

Comment:
try to follow the same idea I explained for add_column test. thanks

Reply:
Thanks for advice regarding optimization of code. Your some changes related to data types and opt params were pulled by me.